lightning/src/routing/scoring.rs

```diff
@@ -901,6 +901,10 @@ struct ChannelLiquidity {
 	/// Time when the historical liquidity bounds were last modified as an offset against the unix
 	/// epoch.
 	offset_history_last_updated: Duration,
+
+	/// The last time when the liquidity bounds were updated with new payment information (i.e.
+	/// ignoring decays).
+	last_datapoint: Duration,
```

Comment:
Name suggests this is the last datapoint itself. I'd expect something like `last_datapoint_time`.